app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -26,7 +26,7 @@ export const PressGallery: React.FC = () => {
 
   const defaultSettings = React.useMemo<IFilterSettingsModel>(() => {
     return {
-      startDate: `${moment(filterAdvanced.startDate).format('YYYY-MM-DD')}`,
+      // startDate: `${moment(filterAdvanced.startDate).format('YYYY-MM-DD')}`,
```

Comment:
this is causing the error, same thing happens in editor when selecting published start date and trying to fetch with said query

Reply:
I'll have to look at this to see what's going on.  If you test in the Filter admin page you can find out what dates work.